src/common.h

```diff
@@ -127,6 +129,29 @@ static string returnFeatureName(FeatureTypes type) {
 
 }
 
+
+class Cov{
+public:
+    //the vraiables used in the coverage calculation function
+    ofstream coverageOutput;
+    int binStart;
+	int binEnd;
```

Comment:
check the indentation here (not super important but better to fix these things now)